types/react-native__assets/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for @react-native/assets 1.0
+// Project: https://github.com/facebook/react-native#readme
+// Definitions by: Zihan Chen <https://github.com/ZihanChen-MSFT>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from './path-support';
+export * from './registry';
```

Comment:
```suggestion
```
I was wrong on discord, this DT rule should be excluded in config file.
One cannot export non-existing entities here (I've checked the original code now, thx)